style.css

```diff
@@ -0,0 +1,282 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+}
+body {
+  background-color: lightskyblue;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.container {
+  position: relative;
+
+  width: 100%;
+  max-width: 1000px;
+  height: 100vh;
+  min-height: 1000px;
+
+  margin: 50px 0;
+
+  background: #fff;
+
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+
+  box-shadow: 0 35px 55px rgba(0, 0, 0, 0.1);
+}
+.side-one {
+  position: relative;
+
+  background: #003147;
+
+  padding: 10px 40px;
+}
+.side-two {
+  position: relative;
+
+  background: #fff;
+
+  padding: 40px;
+}
+.img-bx img {
+  height: 230px;
+  width: 230px;
+
+  border-radius: 125px;
+}
+.img-bx {
+  text-align: center;
+
+  padding-bottom: 20px;
+
+  border-bottom: 2px solid #555;
+}
+.content {
+  display: flex;
+}
+.icon {
+  margin-right: 40px;
+  color: lightskyblue;
+}
+.contact-bx h3 {
+  margin: 10px 0;
+}
+li {
+  list-style-type: none;
+
+  padding: 5px 0;
+}
+.Education-bx h3 {
```

Comment:
it's a good practice to have consistent CSS class names. Here E is in upper case.

Reply:
ok bhai